docs/sources/write/links/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Links
+description: Understand how to link between pages.
+weight: 600
+aliases:
+  - /docs/writers-toolkit/write/links/
+  - /docs/writers-toolkit/write/references/
+  - /docs/writers-toolkit/writing-guide/references/
+keywords:
+  - Hugo
+  - link
+  - linking
+  - links
+  - ref
+  - references
+  - relref
+---
+
+# Links
+
+Choose your link type based on the applicable scenario:
+
+- [Linking from source content that's used (or mounted) in multiple projects](#source-content-is-reused-in-multiple-projects)
+- [Linking to grafana.com pages](#destination-page-is-on-grafanacom)
+- [Linking to external pages](#destination-page-is-external)
+- [Linking to page headings](#anchors)
+
+Other link types exist in our documentation, but you shouldn't use them:
+
+- [Hugo `relref` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#relref)
+
+## Source content is reused in multiple projects
+
+Use the `docs/reference` shortcode.
+
+The source is reused as described in [Reuse directories of content with Hugo mounts](https://grafana.com/docs/writers-toolkit/write/reuse-content/reuse-directories/).
+For more information and examples, refer to [`docs/reference` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference).
+
+## Destination page is on grafana.com
+
+Use a fully qualified URL with version substitution syntax (if needed).
+Version substitution is necessary for fully qualified URLs to link to the correct version of documentation.
+Usually, this is the current version of documentation.
+
```

Comment:
Suggest saying this once because I think this example can be understood to apply to all versioned docs vs repeating it twice when it's not something you do right when you're setting the link most of the time.
````suggestion

In versioned documentation, ensure that you set the appropriate version in the root `_index.md` file for your documentation.
For example, the following YAML, merged with the existing front matter in the root `_index.md` file sets `GRAFANA_VERSION` to be `latest` for that page and all child pages.

  ```yaml
  cascade:
    GRAFANA_VERSION: latest
  ```
````